src/Components/AddRide.js

```diff
@@ -31,27 +31,27 @@ class AddRide extends Component{
     }
     validation = ()=>{
         if(this.state.title.length===0){
-            this.setState({popup:'Ве молиме внесете опис'})
+            this.setState({popup:'Enter description'})
             return false;
         }
         if(this.state.title.length>500){
-            this.setState({popup:'Описот несмее да биде над 500 букви '+this.state.title.length+'/500'})
+            this.setState({popup:'Description must not be more than 500 characters '+this.state.title.length+'/500'})
             return false;
         }
         if(isNaN(this.state.hour)){
-            this.setState({popup:'Ве молиме внесете валиден број во полето за часови'})
+            this.setState({popup:'Enter a valid number in the hour field'})
```

Comment:
I feel like 'please enter' would be better. I might create a 'politeness' branch later on :)

Reply:
Yeah good point, probably a good idea to have a text style guide we can work to. Tried to keep it consistent, but for a first past there are bound to be inconsistencies.

Updated the commit.

Reply:
its maybe something that we can decide on once we can see the actual site from a user perspective